front/style/pages/_citizen-projects.scss

```diff
@@ -290,10 +290,12 @@
         padding: 10px;
     }
 
-    &-form {
+    &__form {
```

Comment:
There are templates that continue to use the old class `citizen-project-form`. You should modify them.